src/Integration.Vsix/Commands/ProjectTestPropertySetCommand.cs

```diff
@@ -85,7 +82,7 @@ protected override void QueryStatusInternal(OleMenuCommand command)
                                           .GetSelectedProjects()
                                           .ToList();
 
-            if (projects.Any() && projects.All(x => ProjectToLanguageMapper.GetLanguageForProject(x).IsSupported))
+            if (projects.Any() && projects.All(x => projectToLanguageMapper.GetAllBindingLanguagesForProject(x).All(l => l.IsSupported)))
```

Comment:
Another logic question (here, and in line 64 above) - should this be `All` or `Any`?
As it happens, both currently work because of the way `GetAllBindingLanguagesForProject` currently works, but the callers shouldn't be relying on that.

A helper method would avoid the problem of having to get the logic right in every caller.